app/src/main/java/com/wednesday/bankingdetails/activity/MainActivity.kt

```diff
@@ -30,8 +44,43 @@ class MainActivity : AppCompatActivity() {
             )
         )
 
-        bankListAdapter = BankListAdapter(bankList)
-        recyclerView.adapter = bankListAdapter
+        getFavouriteBanks()
+    }
+
+    private fun initAdapter(): BankListAdapter {
+        return BankListAdapter(bankList)
+    }
+
+    private fun getFavouriteBanks() {
```

Comment:
I think you should change the name of the this function or remove it altogether. A function is a reusable piece of code but everything should not be in a function. If you use it in one place it shouldn't be a function.

Reply:
Also, the name `getFavoriteBanks` means you will return an array but this doesn't do that. Please rename.

Reply:
This piece of code used more than once so moved into the function.

Reply:
Renamed it